.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+#### Problem Description
+[Describe the issue and explain **why** it is a problem or what the expected output should be]
+
+#### Minimal Reproducible Example
+[Make it simple, copy-pastable and include any library calls]
+
+```R
+
+```
+
+[Include the Image(s) if any, of the Minimal Reproducible Example]
+
+[Include error messages if any, from the Minimal Reproducible Example]
+```
+
+```
+
+#### Version Information
+[This is the output of `packageVersion('ggplot2')`, make sure you have the latest version installed.
+ If comparing against an older version, mention that in the problem description]
+
+#### Remove any sections that do not apply
+[Remove this section too and the one below and remove all the hints in the square brackets]
+
+#### Before you submit
+- [How to create a Minimal Reproducible Example?](https://github.com/tidyverse/ggplot2/blob/master/CONTRIBUTING.md#issues)
+- If you have a question, ask at the [user mailing list](https://groups.google.com/forum/#!forum/ggplot2) or the [developer mailing list](https://groups.google.com/forum/#!forum/ggplot2-dev).
+- [Think about a pull request](https://github.com/tidyverse/ggplot2/blob/master/CONTRIBUTING.md#pull-requests).
```

Comment:
I think you should remove this. I'd prefer not to encourage pull requests until I've had a chance to say whether it's likely to get accepted or not.